_includes/app/txn-sample.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/lib/pq"
+)
+
+func transferFunds(tx *sql.Tx, from int, to int, amount float64) error {
+	// Read the balance.
+	var fromBalance float64
```

Comment:
So I know I'm missing the forest for the trees here but...
I think enough readers will be thrown by the usage of a float for money, even in a fake example, that it could distract from the intended demo.